### ISSUE TITLE
Unknown option "snapshot_preserve_min"

### ISSUE THREAD

**Issue by huegelc** [digint/btrbk#167]
Hi,

something is wrong with my config file and I cant figure out what the issue is

```
btrbk --version
btrbk command line client, version 0.22.2
```

It doesnt work like this as pointed out in README.md:

```
# test config
snapshot_preserve_min   1d
volume ssh://xxx.xxx.xxx.xxx/var/lib/lxd/storage-pools/default/containers
  subvolume test
    ssh_user root
    target send-receive    /storage/lxd/backups
```

or like this:

```
# test config
volume ssh://xxx.xxx.xxx.xxx/var/lib/lxd/storage-pools/default/containers
  subvolume test
    snapshot_preserve_min   1d
    ssh_user root
    target send-receive    /storage/lxd/backups
```

Ideally the second config snippet would make more sense if you have multiple subvolumes. Thus you can set different retention for snapshot but its failing with:

`ERROR: Unknown option "snapshot_preserve" `

Thx

**Comment by digint** [comment]
You are using an ancient version. There has been a change in the configuration file semantics in btrbk-v0.23.0: `snapshot_preserve_min` was not available in v0.22.2.

So, either update to the latest version (recommended!), or read the archived documentation at: https://digint.ch/btrbk/doc/archive/btrbk-0.22.2/readme.html.

**Comment by huegelc** [comment]
Well, I installed it through apt:

```
Package: btrbk
Versions:
0.22.2-1 (/var/lib/apt/lists/de.archive.ubuntu.com_ubuntu_dists_xenial_universe_binary-amd64_Packages) (/var/lib/dpkg/status)
 Description Language:
                 File: /var/lib/apt/lists/de.archive.ubuntu.com_ubuntu_dists_xenial_universe_binary-amd64_Packages
                  MD5: ec54ea71f2f78e1f062befd834da27d0
 Description Language: en
                 File: /var/lib/apt/lists/de.archive.ubuntu.com_ubuntu_dists_xenial_universe_i18n_Translation-en
                  MD5: ec54ea71f2f78e1f062befd834da27d0


Reverse Depends:
Dependencies:
0.22.2-1 - perl (0 (null)) libdate-calc-perl (0 (null)) btrfs-tools (2 3.14) openssh-client (0 (null))
Provides:
0.22.2-1 -
Reverse Provides:

```
So, apparently the package isn´t maintained anymore.
